src/components/nodes/pages/summer-course/summer-course-page.tsx

```diff
@@ -124,7 +130,14 @@ const SummerCoursePage = ({node, ...props}: Props) => {
           <FavoritesList isDisplayOnly />
         </div>
       </div>
-      <div>{/* Related Courses Cards */}</div>
+      <div className="centered">
+        <RelatedCourses
```

Comment:
Since this is a server component, let's fetch the site settings there instead of the course page

Reply:
Hmm for some reason I keep getting an error when I pull the site settings directly into the Related Courses component. I'll continue to look into that issue later this afternoon 🤔